static/styles/right_sidebar.css

```diff
@@ -98,13 +100,31 @@
     overflow: hidden;
 
     .user-name {
+        max-width: 130px;
         overflow: hidden;
         text-overflow: ellipsis;
     }
 }
 
+.first-line {
+    display: flex;
+    position: absolute;
+    top: 10px;
+}
+
+.with-status {
+    position: absolute;
+    top: 0;
+}
+
+.second-line {
+    position: absolute;
+    top: 20px;
+}
```

Comment:
These CSS classes should be scoped inside `#user_presences` or something similar -- they're too generic names to be in global scope.